AAD.fs/common.props

```diff
@@ -0,0 +1,12 @@
+<Project>
+  <PropertyGroup>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>    
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
Where is this used?

Reply:
It is [documented here](https://github.com/dotnet/sourcelink/tree/master/docs#continuousintegrationbuild). It is for deterministic, and will work once https://github.com/dotnet/fsharp/issues/9379 is fixed or I find a workaround. I think we can leave it in.